sql/core/src/test/resources/sql-tests/inputs/join.sql

```diff
@@ -0,0 +1,225 @@
+-- join nested table expressions (auto_join0.q)
```

Comment:
Do we need to reference to the hive `.q` file? I think hive golden file tests will be removed eventually.